src/locales/en/translation.json

```diff
@@ -786,16 +786,19 @@
       "header": "Update your name or date of birth",
       "titleCoi": "Update your full name or date of birth",
       "headerCoi": "Update your full name or date of birth",
+      "titleRepeatFraudCheck": "Update your full name or date of birth",
+      "headerRepeatFraudCheck": "Update your full name or date of birth",
```

Comment:
As these are the same, can we just drop the context for these fields?

Reply:
Although do we ever use this page _without_ context? If not we can drop the context versions of the content and just use `title` and `header` (after updating them).

Reply:
It's used for the current 6MFC journey (which doesn't allow you to update any name or DOB).

But once COI is live, we could make that simplification :)